### ISSUE TITLE
sorting of products in card and emails

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#196]
order products in card and emails by
- first: product_order
- second: productVariantValue

As a result the Day-Tickets should be sorted like:
- August 4
- August 5
- August 6

![screen3](https://cloud.githubusercontent.com/assets/1533081/6671677/38a19860-cc07-11e4-93a4-37794e62816f.png)

**Comment by inbaz** [comment]
I think it's not possible to sort for the content. Are you fine with sorting by product variant value order?

**Comment by smadamit** [comment]
Yes sorting by productVariantValue-order is what I really meant.
